src/components/data_search/DatasetFilterList.jsx

```diff
@@ -64,6 +65,27 @@ export const DatasetFilterList = (props) => {
           })
         }
       </List>
+      <Typography variant="h6" gutterBottom component="div" sx={{ fontFamily: 'Montserrat' }} marginTop="1em">
```

Comment:
Could these elements be refactored to avoid code duplication? They look pretty similar to me.

Reply:
I definitely want to do this long term, let me see how much work it would be.

Reply:
It wasn't too challenging, I created two new components.